src/main/java/uk/gov/hmcts/cmc/claimstore/services/CountyCourtJudgmentService.java

```diff
@@ -0,0 +1,83 @@
+package uk.gov.hmcts.cmc.claimstore.services;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+import uk.gov.hmcts.cmc.claimstore.events.EventProducer;
+import uk.gov.hmcts.cmc.claimstore.exceptions.ForbiddenActionException;
+import uk.gov.hmcts.cmc.claimstore.exceptions.NotFoundException;
+import uk.gov.hmcts.cmc.claimstore.models.Claim;
+import uk.gov.hmcts.cmc.claimstore.processors.JsonMapper;
+import uk.gov.hmcts.cmc.claimstore.repositories.ClaimRepository;
+
+import java.time.LocalDate;
+import java.util.Map;
+
+@Component
+public class CountyCourtJudgmentService {
+
+    private final ClaimRepository claimRepository;
+    private final JsonMapper jsonMapper;
+    private final EventProducer eventProducer;
+
+    @Autowired
+    public CountyCourtJudgmentService(
+        ClaimRepository claimRepository,
+        JsonMapper jsonMapper,
+        EventProducer eventProducer) {
+        this.claimRepository = claimRepository;
+        this.jsonMapper = jsonMapper;
+        this.eventProducer = eventProducer;
+    }
+
+    @Transactional
+    public Claim save(final long submitterId, final Map<String, Object> data, final long claimId) {
+
+        Claim claim = getClaim(claimId);
+
+        if (!isClaimSubmittedByUser(claim, submitterId)) {
+            throw new ForbiddenActionException("It's not your claim");
+        }
+
+        if (isResponseAlreadySubmitted(claim)) {
+            throw new ForbiddenActionException("Response for the claim was submitted");
```

Comment:
The same here, at least saying the claim ID would give more value.

Reply:
done